hedera-node/src/test/java/com/hedera/services/store/tokens/HederaTokenStoreTest.java

```diff
@@ -1754,22 +1749,33 @@ void rejectsNegativeFractionInFractionalFee() {
 
 	@Test
 	void rejectsNegativeAmountInFixedFee() {
-		var req = fullyValidAttempt().setCustomFees(grpcNegativeFixedCustomFees).build();
+		final var req = fullyValidAttempt().setCustomFees(grpcNegativeFixedCustomFees).build();
 
 		// when:
-		var result = subject.createProvisionally(req, sponsor, CONSENSUS_NOW);
+		final var result = subject.createProvisionally(req, sponsor, CONSENSUS_NOW);
 
 		// expect:
 		assertEquals(CUSTOM_FEE_MUST_BE_POSITIVE, result.getStatus());
 		assertTrue(result.getCreated().isEmpty());
 	}
 
+	@Test
+	void rejectsZeroAmountInFixedFeeOnTokenUpdate() {
```

Comment:
nit : `rejectsNonPositiveAmountInFixedFeeOnTokenUpdate` would be a better name

Reply:
I disagree. This scenario is indeed for zero fixed custom fee. The negative fixed fee was tested with TokenCreate already. So with this test added, we have full validations on the fixed fee.